src/main/java/com/mentorship/vineservice/controllers/exeptions/ErrorResponse.java

```diff
@@ -0,0 +1,41 @@
+package com.mentorship.vineservice.controllers.exeptions;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+@Setter
+public class ErrorResponse {
+
+    private HttpStatus status;
+    private LocalDateTime timeStamp;
+    private List<String> errors;
+
+
+    public ErrorResponse(HttpStatus status) {
+        this();
+        this.status = status;
+    }
+
+
+    public ErrorResponse(HttpStatus status, List<String> errors) {
+        this();
```

Comment:
I think that u can initialize timestamp straight away instead of calling constructor